src/encoder.rs

```diff
@@ -2414,7 +2404,7 @@ fn encode_partition_bottomup(
             // two partitioned rectangles, defined in 'partitions', of the current block
             // is passed to encode_partition_bottomup()
             for offset in partitions {
-                if let (cost, Some(mode_decision)) = encode_partition_bottomup(
+                let rdo_output = encode_partition_bottomup(
```

Comment:
I expect that using the same name here as in line 2400 can confuse readers.
So, I am changing its name in this scope as 'child_rdo_output'.